marlowe-playground-client/src/MainFrame/Types.purs

```diff
@@ -195,6 +217,12 @@ newtype State
   -- as their state is part of the MainFrame state, but we cannot inspect the state of the
   -- child components Blockly and ActusBlockly as we need a Query.
   , hasUnsavedChanges :: Boolean
+  -- The initial language selected when you create/load a project indicates the workflow a user might take
+  -- A user can start with a haskell/javascript example that eventually gets compiled into
+  -- marlowe/blockly and runned in the simulator, or can create a marlowe/blockly contract directly,
+  -- which can be used interchangeably. This is used all across the site to know what are the posible
+  -- transitions.
```

Comment:
s/runned/run

Reply:
What does `Nothing` represent?

Reply:
1 - replaced
2 - flow not selected. For example if you just started or if you return to home (forgot to add the second one, just added it)